[Docs] Improve the instructions/documentation in the README

psobolewskiPhD opened the pull request:
First, thanks for the great extension!
I've updated and clarified the instructions or prediction and training in the README based on recent image.sc threads and my own helping of end-users.
I moved up the Prediciton instructions, because they are simpler and can be used for training. I also provide some info on running the model on H&E style images. Finally, I clarify the training section and also inform that human-in-the-loop style training is also possible by alternating between Prediction and Training.

lacan commented on README.md:
```diff
@@ -16,7 +16,7 @@ they can play nice.
+## Step 1: Install Cellpose and/or Omnipose
 
 ## Installation with Conda/Mamba
 We provide the following YAML file that installs Cellpose and omnipose in the same environment.
```
Perhaps we should remove this whole part, as itàs bad for mac and the yaml is out of date.

psobolewskiPhD replied:
Makes sense, I think that with CUDA etc. installation is tricky enough that pointing people at the upstream docs might be easiest. Can just stress that an env with working cellpose is needed.

lacan commented on README.md:
```diff
@@ -54,7 +58,8 @@ F:\conda-envs\cellpose-omnipose-biop-gpu\python.exe
 > **Note**
-> While this example is done under Windows, this will work on Mac and Linux as well. 
+> While this example is done under Windows, this will work on Mac and Linux as well. You may need to change
+> `where` to `which`.
```
We can add explicit examples for both Win and Mac/Linux so that people see examples of what it looks like

psobolewskiPhD replied:
Sure. 
I just tested on my Mac and my linux VM (ubuntu 23.10) in Bash shell `where` doesn't work. 
In `zsh` the default shell on macOS now, `where` does work. Default Ubuntu doesn't have zsh.
`which` works in both shells and both platforms.
I'll edit this section shortly.

lacan reviewed (changes requested):
Hi @psobolewskiPhD and thanks for taking the time to update the documentation, I really appreciate it. 

I've offered a couple of comments. If you are OK with them and to address them I will merge this ASAP. 

All the best
Oli

psobolewskiPhD commented:
@lacan Sure! Feel free to push to this branch as you wish--should have edit rights. 
❤️ 
I can add example of checking the path in macOS/Linux.

psobolewskiPhD commented:
@lacan I think I addressed the comments, feel free to edit.
I also switched the formatting to use the nicer (colorful) Markdown alerts. I think less grey is nice, makes the text more readable on a white background.

psobolewskiPhD commented on README.md:
```diff
@@ -67,23 +70,25 @@ To copy `run-cellpose-qc.py`, go to Extensions > Installed Extensions and click
 
 Go to `Edit > Preferences > Cellpose/Omnipose`
-Complete the fields with the requested information. based on the `mamba` installation above, this is what it should look like:
+Complete the fields with the requested information. based on the `mamba` installation above, this is what it should look like on Windows:
```
```suggestion
Complete the fields with the requested information by pasting the path(s) you obtained above. 
Based on the `mamba` installation above, this is what it should look like on Windows:
```

lacan reviewed (approved):
Awesome, thank you!
